routemaster/state_machine/actions.py

```diff
@@ -88,6 +94,12 @@ def _process_action_with_metadata(
     return True
 
 
+def _calculate_idempotency_token(label: LabelRef, history_id: int):
+    return hashlib.sha256(
+        f"{label.name}:{history_id}".encode('ascii'),
```

Comment:
I don't believe `name` is restricted to being ASCII elsewhere; would advocate for encoding this in UTF-8.

Reply:
Yeah I just dropped the label name. History IDs are unique. 3e7b914b5b2078ccc710e251a560b59b1c3cee15